compute/instances/how-to/use-boot-modes.mdx

```diff
@@ -133,6 +133,30 @@ Follow the instructions below to reboot an existing Instance using cloud-init.
 
 Your Instance is rebooted using Cloud-init.
 
+## How to change your boot volume
+
+The [boot-on-block](/concepts#boot-on-block) feature allows Instances to boot from attached [Block Storage](/concepts#block-volumes) volumes instead of local volumes. When [creating an Instance](/how-to/create-an-instance), you can select Block Storage as a boot volume. 
+
+Once the Instance is created, you can select a different volume to boot from. 
+
+1. Click **Instances** in the **Compute** section of the side menu. The Instances page displays.
+
+2. Click the Instance you wish to boot into Rescue Mode.
```

Comment:
Is this right? Should it be "Click the Instance you wish to change the boot volume for" ?